perfkitbenchmarker/sample.py

```diff
@@ -33,10 +34,12 @@ def PercentileCalculator(numbers):
   count = len(numbers_sorted)
   total = sum(numbers_sorted)
   result = {}
-  for percentile in PERCENTILES_LIST:
+  for percentile in percentiles:
```

Comment:
thanks for the information about numpy.percentile, we will take a look at that. This percentile calculator was added long time ago, here in this PR we are just adding an option to allow it to output more/finer-grain percentiles.

The potential work to replace this percentile calculator with  numpy.percentile can be tracked separately. We need to make sure it can do what we want do here (the advantage of having our own percentile calculator is we can add any new feature/functionalities to it as necessary).